src/configuration/external/nodesecure/index.ts

```diff
@@ -1,16 +1,34 @@
+// Node.Js Dependencies
+import { readFile } from "fs/promises";
+import { join, dirname } from "path";
+import { fileURLToPath } from "url";
+
 // Import Third-party Dependencies
 import { RC as NodeSecureRuntimeConfig, read } from "@nodesecure/rc";
 import { match } from "ts-pattern";
 import type { Result } from "ts-results";
 
 // Import Internal Dependencies
+import { consolePrinter } from "../../../../lib/console-printer";
 import { Maybe } from "../../../types/index.js";
 import {
   defaultExternalConfigOptions,
   ExternalConfigAdapter,
   ExternalRuntimeConfiguration
 } from "../common.js";
 
+import {
+  validateIgnoreFile,
+  kIgnoreFileName,
+  IgnorePatterns,
+  IgnoreWarningsPatterns
+} from "./ignore-file";
+
+const { font: log } = consolePrinter;
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const kRootPath = join(__dirname, "..", "..", "..", "..");
+export const kIgnoreFilePath = join(kRootPath, kIgnoreFileName);
```

Comment:
Is there any particular reason here to not use `process.cwd()` to access the root dir?

Imho it would provide a more consistent way (refactoring safe) of accessing root files such as:
```suggestion
export const kIgnoreFilePath = join(process.cwd(), kIgnoreFileName);
```

Reply:
Don't even know myself haha 🙈